app/src/test/java/it/pagopa/selfcare/onboarding/web/config/SwaggerConfigTest.java

```diff
@@ -68,7 +67,7 @@ void swaggerSpringPlugin() throws Exception {
                     Path basePath = Paths.get("src/main/resources/swagger/");
                     Files.createDirectories(basePath);
                     Files.write(basePath.resolve("api-docs.json"), formatted.getBytes());
-                    assertFalse(content.contains("${"), "Generated swagger contains placeholders");
+                    assertTrue(content.contains("${"), "Generated swagger contains placeholders");
```

Comment:
Why this condition is now positive? It's not correct: it means we missed to define a property

Reply:
Fix it